CONTRIBUTING.md

```diff
@@ -43,6 +43,30 @@ When creating an issue, please ensure you include the following information
 6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
 7. When all PRs are completed the issue is then closed.
 
+## Documenting your change
+
+## what to document
+
+Document changes if:
+
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+## how to document
+
+Docs work in:[link](https://github.com/rackerlabs/docs-rpc "docs-rpc")
```

Comment:
This link is weird.

Maybe:

The doc repo is [here](https://github.com/rackerlabs/docs-rpc)